internal/controller/bootstrap/controlplane_bootstrap_controller.go

```diff
@@ -19,40 +19,34 @@ package bootstrap
 import (
 	"context"
 	"fmt"
-	kutil "github.com/k0sproject/k0smotron/internal/util"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"sigs.k8s.io/cluster-api/controllers/remote"
-	"sigs.k8s.io/cluster-api/util/kubeconfig"
 	"strings"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
pkg/errors is deprecated/archived. Use `fmt.Errorf("foo: %w", err)` instead of `errors.Wrap(err, "foo")`.

Reply:
Was late